cirq_superstaq/ops/qubit_gates.py

```diff
@@ -753,6 +753,75 @@ def _from_json_dict_(cls, **kwargs: Any) -> "IXGate":
 AQTICCX = AQTITOFFOLI = IX.controlled(2, [0, 0])
 
 
+@cirq.value_equality(approximate=True)
+class StrippedCZGate(cirq.Gate):
+    """The Stripped CZ Gate is the gate that is actually being performed by Hilbert,
+    which is just a regular CZ gate when phi = 0, and is corrected into a CZ
+    gate by RZ gates afterwards if phi is nonzero.
+    """
+
+    def __init__(self, phi: cirq.TParamVal = 0) -> None:
```

Comment:
can we use a more descriptive variable name? e.g. something like
```suggestion
    def __init__(self, rz_rads: cirq.TParamVal = 0) -> None:
```

Reply:
I would support making that both here and everywhere else in the class.

Follow-up proposal: maybe we should make this parameter something like `z_exponent` and make it a multiple of `pi`?

Reply:
I think I'll do the rz_rads for now, and then if/when we change ParallelRGates to the multiples of pi format I'll change it here too.